.pre-commit-config.yaml

```diff
@@ -1,124 +1,129 @@
 repos:
+  # Quick content checks based on grepping for python specific patterns:
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.9.0
+    hooks:
+      - id: python-check-blanket-noqa # Prohibit overly broad QA exclusions.
+      - id: python-no-eval # Never use eval() it's dangerous.
+      - id: python-no-log-warn # logger.warning(), not old .warn()
+      - id: rst-backticks # Find single rather than double backticks
+      - id: rst-directive-colons # Missing double-colons after directives
+      - id: rst-inline-touching-normal # Inline code should never touch normal text
+      - id: python-use-type-annotations # Use annotations not old-style type comments
```

Comment:
If we've got different autoformatting set up for YAML files in different editors, we're going to get a ton of changes going back and forth, and potentially a lot of meaningless merge conflicts. Can we standardize how YAML gets formatted and integrate that into the repo somehow?

Reply:
There is a [yaml format pre-commit hook](https://github.com/jumanjihouse/pre-commit-hook-yamlfmt#on-demand) we could set up. I had a hard time nailing down the correct config to math our existing format so I'd like to leave it for another PR.

Reply:
I opened up a [PR](https://github.com/catalyst-cooperative/pudl/pull/1689) that uses a yaml formatter pre-commit to address this issue.